MAINTAINER.md

```diff
@@ -0,0 +1,15 @@
+# Instructions for maintainers of the nodereport project
+
+## Releasing to www.npmjs.com

```

Comment:
Are you sure? (some) People are already confused aboutthe difference between npm the cli, and npmjs.org the package repository... but above should be `npmjs.org`, IMO, not the company URL.